modules/AOS_PDF_Templates/templateParser.php

```diff
@@ -110,11 +114,43 @@ public static function parse_template_bean($string, $key, &$focus)
                         $link = $secureLink;
                         $repl_arr[$key . "_" . $fieldName] = '<img src="' . $link . '" width="' . $field_def['width'] . '" height="' . $field_def['height'] . '"/>';
                     }
-                } elseif ($field_def['type'] == 'wysiwyg') {
+                } elseif ($fieldType === 'wysiwyg') {
                     $repl_arr[$key . "_" . $field_def['name']] = html_entity_decode($focus->$field_def['name'],
                         ENT_COMPAT, 'UTF-8');
                     $repl_arr[$key . "_" . $fieldName] = html_entity_decode($focus->{$fieldName},
                         ENT_COMPAT, 'UTF-8');
+                } elseif ($fieldType === 'datetime' || $fieldDbType === 'datetime') {
+                    $templateFormat = $template->template_date_format ?? '';
+
+                    $datetimeValue = $focus->{$fieldName} ?? '';
+                    $templateFormat = $template->template_date_format ?? '';
```

Comment:
Hi @jack7anderson7,

We have a duplicate `templateFormat = $template->template_date_format ?? '';`